src/cascadia/TerminalConnection/CTerminalHandoff.cpp

```diff
@@ -101,8 +103,10 @@ HRESULT CTerminalHandoff::EstablishPtyHandoff(HANDLE in, HANDLE out, HANDLE sign
     RETURN_IF_FAILED(_duplicateHandle(in, in));
     RETURN_IF_FAILED(_duplicateHandle(out, out));
     RETURN_IF_FAILED(_duplicateHandle(signal, signal));
-    RETURN_IF_FAILED(_duplicateHandle(process, process));
+    RETURN_IF_FAILED(_duplicateHandle(ref, ref));
+    RETURN_IF_FAILED(_duplicateHandle(server, server));
+    RETURN_IF_FAILED(_duplicateHandle(client, client));

```

Comment:
It seems like we are duplicating the handles *twice*. Is that so? Once here and once in PackPseuedo

Reply:
Yeah it's probably too many times but I haven't fully shaken that out yet. Do you want me to pull one and risk it or just ride with it for now?

Reply:
Oh hm I guess I have leftovers if I do it this way. OK. I'll remove it from pack.